plugins/profile.py

```diff
@@ -182,8 +182,7 @@ async def set_profile_picture(message: Message):
             "-lname": "Print only last name",
             "-flname": "Print full name",
             "-bio": "Print bio",
-            "-uname": "Print username",
```

Comment:
Why you deleted "-uname": "Print username",

Reply:
Its still there 🤔